src/services/ui/package.json

```diff
@@ -15,9 +15,13 @@
   },
   "dependencies": {
     "@aws-amplify/auth": "^5.4.0",
+    "@cmsgov/design-system": "2.7.1",
```

Comment:
im sure onemac is using this library for styling now, but Id like to see if we can accomplish this without using this package. in this project we are using a combination of tailwind and the macpro-ux-lib.